pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
+
+import { Btn } from "@/components/Button";
+
+import React, { useState } from "react";
+
+import myComponent from "@/components/myComponent1/myComponent.js";
+
+const kosmetikProdukt = [
+  {
+    name: "Luminous Silk Foundation",
+    brand: "Aurora",
+    type: "Foundation",
+    shade: "Porcelain",
+    benefits: ["hydrating", "long-lasting", "lightweight"],
+    price: "45€",
+    size: "30ml",
```

Comment:
I would suggest to list size (no string) and unit separately for better calculation.